src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -111,222 +35,170 @@ export const TransactionGenericRow = ({
   status,
   title,
   transfersByType,
-  fee,
-  txid,
   txData,
-  blockTime,
-  txLink,
-  compactMode = false,
   toggleOpen,
-  parentWidth,
 }: TransactionGenericRowProps) => {
-  const {
-    columns,
-    dateFormat,
-    breakPoints: [isLargerThanLg],
-  } = GetTxLayoutFormats({ parentWidth })
-
   const txMetadataWithAssetId = useMemo(() => getTxMetadataWithAssetId(txData), [txData])
 
-  const transfers = useMemo(() => {
-    return Object.values(transfersByType).map((transfersOfType, index) => {
-      const hasManyTypeTransfers = transfersOfType.length > 1
-
-      return (
-        <React.Fragment key={index}>
-          {hasManyTypeTransfers ? (
-            <AssetsTransfers index={index} compactMode={compactMode} transfers={transfersOfType} />
-          ) : (
-            <AssetTransfer index={index} compactMode={compactMode} transfer={transfersOfType[0]} />
-          )}
-        </React.Fragment>
-      )
-    })
-  }, [compactMode, transfersByType])
-
-  const isNft = useMemo(() => {
-    return Object.values(transfersByType)
-      .flat()
-      .some(transfer => !!transfer.id)
-  }, [transfersByType])
-
-  const cryptoValue = useMemo(() => {
-    if (!fee) return '0'
-    return fromBaseUnit(fee.value, fee.asset.precision)
-  }, [fee])
-
-  const fiatValue = useMemo(() => {
-    return bnOrZero(fee?.marketData?.price)
-      .times(cryptoValue)
-      .toString()
-  }, [fee?.marketData?.price, cryptoValue])
-
-  const gridTemplateColumns = useMemo(() => ({ base: '1fr', md: columns }), [columns])
-  const iconCircleBoxSize = useMemo(
-    () => ({ base: '24px', lg: compactMode ? '24px' : '38px' }),
-    [compactMode],
-  )
-  const stackDirection: StackDirection = useMemo(
-    () => ({ base: 'row', md: 'column', xl: compactMode ? 'row' : 'column' }),
-    [compactMode],
+  const hasManySends = useMemo(
+    () => transfersByType.Send && transfersByType.Send.length > 1,
+    [transfersByType.Send],
   )
-  const stackFontSize = useMemo(
-    () => ({ base: 'sm', lg: compactMode ? 'sm' : 'md' }),
-    [compactMode],
+  const hasSendAndRecieve = useMemo(
+    () =>
+      transfersByType.Send &&
+      transfersByType.Send.length > 0 &&
+      transfersByType.Receive &&
+      transfersByType.Receive.length > 0,
```

Comment:
Could only do `transfersByType.Send?.length && transfersByType.Receive?.length` for conciseness